fusor-ember-cli/app/styles/custom.scss

```diff
@@ -864,6 +864,10 @@ span.hanging-indent{
   padding-bottom: 16px;
 }
 
+.cancel-back-next-row {
+  margin-top: 67px;
```

Comment:
height of cancel-back-next buttons + padding + border so that whatever was above the fixed position button bar will not be hidden by it.